lib/utils/querySelectorHelper.js

```diff
@@ -232,12 +237,29 @@ function* processSelectors(element, selectors) {
     }
 
     if (rule.hasOwnProperty('attrs')) {
-        if (!rule.attrs.every(function (attr) {
+        if (!rule.attrs.some(function (attr) {
+            if (!element.hasAttribute(attr.name)) {
+                return false;
+            }
+
+            var value = element.getAttribute(attr.name);
```

Comment:
use let

if you want to set value here, you could remove undefined from the switch and add a if before:

```js
if (attr.operator === undefined) {
    return element.hasAttribute(attr.name);
}
```

Because for this case we don't need the value